build.gradle

```diff
@@ -43,19 +43,10 @@ springBoot {
     mainClass = xhAppPackage + ".Application"
 }
 
-if (parseBoolean(runHoistInline)) {
```

Comment:
Personal ignorance only - what does `settings.gradle` do which we did not get here?